web/src/components/Meditor.vue

```diff
@@ -148,7 +149,6 @@
             class="px-7 py-5 overflow-y-auto"
           >
             <v-jsf
-              ref="vjsfRef"
```

Comment:
Why was this ref removed? Seems unrelated

Reply:
Sorry, I forgot to mention that, it probably should've been in a separate PR. It's an unused variable.